test/instrumentation/modules/tedious.test.js

```diff
@@ -1,11 +1,7 @@
 'use strict'
 
-// Don't test on travis, there is no mssql service
-if (process.env.TRAVIS) process.exit()
```

Comment:
Last mention of travis in this repo's active code, leaving only a few stray comments. [test/instrumentation/modules/restify/basic.test.js](https://github.com/elastic/apm-agent-nodejs/blob/cf2b0c59bfde974ff5ad2403f9fc86f9b3a502fe/test/instrumentation/modules/restify/basic.test.js#L53)

Reply:
Cool